src/block/styled-components.js

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import {styled} from '../styles/index';
+import type {Font} from '../styles/types';
+import type {StyledBlockPropsT} from './types';
+
+export const StyledBlock = styled(
+  'div',
+  ({
+    $theme,
+    $color,
+    $font,
+    $alignContent,
+    $alignItems,
+    $alignSelf,
+    $direction,
+    $display,
+    $flex,
+    $justifyContent,
+    $justifySelf,
+    $position,
+    $width,
+    $minWidth,
+    $maxWidth,
+    $height,
+    $minHeight,
+    $maxHeight,
+    $overflow,
+    $margin,
+    $marginTop,
+    $marginRight,
+    $marginBottom,
+    $marginLeft,
+    $padding,
+    $paddingTop,
+    $paddingRight,
+    $paddingBottom,
+    $paddingLeft,
+    $wrap,
+    $left,
+    $top,
+    $right,
+    $bottom,
+    ...other
+  }: StyledBlockPropsT) => {
+    const {colors, typography, sizing} = $theme;
+
+    let styles = {};
+
+    if ($color !== undefined) {
+      styles.color = colors[$color] || $color;
+    }
+
+    if ($font !== undefined) {
+      const font: Font = typography[$font];
+      styles.fontFamily = font.fontFamily;
+      styles.fontWeight = font.fontWeight;
+      styles.fontSize = font.fontSize;
+      styles.lineHeight = font.lineHeight;
+    }
+
+    if ($alignContent) {
```

Comment:
@nadiia Open styletron question...

Is it better to just apply these values blindly and let styletron remove them or is it preferred to do an `undefined` as shown here?